src/lib/redux/siteDataSlice.ts

```diff
@@ -9,6 +9,7 @@ interface siteDataSliceState {
   sites: any[];
 }
 
+//TODO @julianrkung: Change sites to siteIdsToSites and change currentSite to currentSiteId
```

Comment:
nit: lacking context, but can I confirm this is a note to refactor s.t the `sites` object is now a map, not an array? (i.e not just renaming)

Reply:
Yes, this is a note to refactor `sites` into an object `siteIdsToSites: Record<SiteId, SiteRecord>` and have `currentSite` become a string `currentSiteId` that will be used to index into `siteIdsToSites`. This is done mostly because we're shifting to using `id` as lookups.

The refactor will also put a wrapper over the `Site` portion of redux store so it's easier to interface with, so adding stuff like `getCurrentSite`, etc...